.github/ubiquibot-config.yml

```diff
@@ -1,5 +1,6 @@
 evm-network-id: 100
 price-multiplier: 1.5
+issue-creator-multiplier: 1
```

Comment:
This should be 2 as given in the issue discussion. https://github.com/ubiquity/ubiquibot/issues/583#issue-1838350837

Reply:
done